base/VulkanglTFModel.cpp

```diff
@@ -6,7 +6,7 @@
  * This code is licensed under the MIT license (MIT) (http://opensource.org/licenses/MIT)
  */
 
-#pragma once
+//#pragma once
```

Comment:
Any reason the pragma once is commented out? Is this required for Mac OSX builds?

Reply:
It was presenting a warning stating that #pragma once should only be used in header files. I believe it was only a warning. I may have been overzealous. Would you like to keep it in? I can check it will build.

Reply:
That's why I commented it out instead of deleting it. I wasn't sure why you had it in the file, but thought there might be a reason on another platfrom.

Reply:
You're indeed correct. Pragma once doesn't sake sense outside of headers. I guess this is a left-over from splitting the model loader into a header + unit.